package.json

```diff
@@ -4,7 +4,7 @@
   "description": "CLI tool to run BigCommerce Stores locally for theme development.",
   "main": "index.js",
   "engines": {
-    "node": ">= 8.0.0 <9.0.0"
+    "node": ">= 8.0.0 <11.0.0"
```

Comment:
Also can we also update the readme to reflect node version support.

Reply:
Good catch!